custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -143,6 +143,7 @@ def get_person_case_structure(self):
                 'update': {
                     'age': self.patient_detail.page,
                     'age_entered': self.patient_detail.page,
+                    'contact_phone_number': '91' + validate_phone_number(self.patient_detail.pmob),
```

Comment:
what happens if `validate_phone_number` raises an exception when converting to int, or raises an AssertionError?

also could we add validation in `validate_phone_number` to check that the phone number is positive after the int conversion